tests/components/squeezebox/__init__.py

```diff
@@ -35,6 +37,8 @@
     STATUS_SENSOR_RESCAN: 1,
     STATUS_SENSOR_LASTSCAN: 0,
     STATUS_QUERY_LIBRARYNAME: "FakeLib",
```

Comment:
I think all of this really belongs in `conftest.py`; which is where I moved it in the test PR (https://github.com/home-assistant/core/pull/125378).

Reply:
Still my intention to move this all in post, sorry if I haven't communicated that well. I wasn't planning on a update platform but here we are.  

How does this sit with you:-

1. Revert non media player, borwse, init test move in PR (https://github.com/home-assistant/core/pull/125378). @rajlaud 
2. Finish all sensor work, this should be the last for now. @pssc 
3. Finish work on new tests for media player, etc @rajlaud 
4. Fold in sensor platforms to new tests (empty test init etc) @pssc)
5. Continue with other planned PR's...

Reply:
No problem, I already moved everything over except for the new lines related to update. Let's merge in the tests PR first (https://github.com/home-assistant/core/pull/125764). Only a couple of lines will need to change in this PR.